operations/observability/mixins/self-hosted/dashboards/observability/cardinality-management-overview.json

```diff
@@ -239,6 +239,12 @@
             "displayMode": "auto",
             "inspect": false
           },
+          "links": [
+            {
+              "title": "Explore",
+              "url": "https://grafana.gitpod.io/explore?orgId=1&left=%7B%22datasource%22:%22P4169E866C3094E38%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22editorMode%22:%22code%22,%22expr%22:%22%7B__name__%3D%5C%22${__data.fields.metric}%5C%22%7D%22,%22legendFormat%22:%22__auto%22,%22range%22:false,%22instant%22:true,%22exemplar%22:false%7D%5D,%22range%22:%7B%22from%22:%22now-5m%22,%22to%22:%22now%22%7D%7D"
```

Comment:
Shouldn't the `datasource` in this query be templated? I see a specific value of `22P4169E866C3094E38` which I can't make much sense of but it works in prod 🤔

Reply:
The explore page doesn't have a templated datasource, only dashboards :/

Reply:
That's a bummer :/. Considering this dashboard is for the maintainer's usage i.e us, I don't think we should worry about this much but how do you think this works when we rotate prometheus, etc and the id changes? 🤔

Reply:
If we change our datasource, then this needs to be replaced indeed 😕 

After almost 2 years working here, this has never happened though 😅

Reply:
:shipit: then! 👍🏼